Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -583,7 +584,24 @@ public double MainWindowWidth
 
         public string OpenResultCommandModifiers => Settings.OpenResultModifiers;
 
-        public string PreviewHotkey => Settings.PreviewHotkey;
+        public string PreviewHotkey 
+        { 
+            get
+            {
+                // TODO try to patch issue #1755
+                // Should be removed after some time
+                try
+                {
+                    var converter = new KeyGestureConverter();
+                    var key = (KeyGesture)converter.ConvertFromString(Settings.PreviewHotkey);
```

Comment:
probably this is better? [ConvertFromInvariantString(String)](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.typeconverter.convertfrominvariantstring?view=windowsdesktop-7.0#system-componentmodel-typeconverter-convertfrominvariantstring(system-string))

Correct me if i am wrong

Reply:
Probably CultureInfo matters here, we are not dealing with datetime, decimal or floats.